Core/Services/StitchCoreRemoteMongoDBService/Sources/StitchCoreRemoteMongoDBService/Sync/NamespaceSynchronizationConfig.swift

```diff
@@ -59,18 +59,18 @@ internal class NamespaceSynchronization: Sequence {
                 return nil
             }
 
-            return CoreDocumentSynchronization.init(docsColl: docsColl,
-                                                    config: &values[index],
-                                                    errorListener: errorListener)
+            return try? CoreDocumentSynchronization.init(docsColl: docsColl,
```

Comment:
[for further discussion] I don't think we should be squashing errors like this

Reply:
Will do in the error ticket.